common/dynamicconfig/constants.go

```diff
@@ -713,6 +713,11 @@ This config is EXPERIMENTAL and may be changed or removed in a later release.`,
 		10,
 		`FrontendMaxBadBinaries is the max number of bad binaries in namespace config`,
 	)
+	FrontendHideInternalOrUnknownErrors = NewGlobalBoolSetting(
```

Comment:
Why not make it Namespace level config? A namespace level config can still have a default value that apply to all namespaces.

Reply:
will make in namespace in the next rev.

Reply:
done